src/redis_index.c

```diff
@@ -205,6 +205,9 @@ RedisSearchCtx *NewSearchCtxDefault(RedisModuleCtx *ctx) {
 }
 
 void SearchCtx_Free(RedisSearchCtx *sctx) {
+  //  if (sctx->keyName) {
```

Comment:
?

Reply:
I will remove the comment, could not put it here cause it cause issues on other location (which probably needs it)

Reply:
ohhhh i see.. comment with TODO or something

Reply:
not sure it worth doing cause on master we do not have this issue ..